x-pack/test/functional/services/ml/data_visualizer_file_based.ts

```diff
@@ -125,5 +125,13 @@ export function MachineLearningDataVisualizerFileBasedProvider(
         await testSubjects.existOrFail('mlFileImportSuccessCallout');
       });
     },
+
+    async selectCreateFileBeatConfig() {
```

Comment:
Nit - `Filebeat` is one word, so this should be `selectCreateFilebeatConfig` I think.